.gitignore

```diff
@@ -1,3 +1,3 @@
 *.pyc
 .idea/
-package-lock.json
+*.json
```

Comment:
don't we have fake data as json ?

Reply:
anytime we rerun the create fake data it changes thousands of lines in the fake data jsons so we thought we could ignore those for the sake of making commits easier to go through